docs/src/tutorials/bank_fraud_detection_with_tfdf.ipynb

```diff
@@ -0,0 +1,1316 @@
+    "\n",
+    "\n",
+    "*Note: This notebook assumes a basic understanding of Temporian. If you are not familiar with Temporian, we recommend that you read the [3 minutes guide to Temporian](https://temporian.readthedocs.io/en/latest/3_minutes) guide first.*\n",
+    "\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "8ce89356-af4a-4ddb-9f37-5e57ffe7ef5a",
+   "metadata": {},
+   "source": [
+    "## Install and import dependencies\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "77dc817d-b6fa-4b89-af98-efadcfb30324",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Note: you may need to restart the kernel to use updated packages.\n",
+      "Note: you may need to restart the kernel to use updated packages.\n",
+      "Note: you may need to restart the kernel to use updated packages.\n",
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# For data preprocessing and augmantation\n",
+    "%pip install temporian -q\n",
+    "\n",
+    "# For model training\n",
+    "%pip install tensorflow tensorflow_decision_forests -q\n",
+    "\n",
+    "# To plots the results\n",
+    "%pip install seaborn -q\n",
+    "\n",
+    "# To compute the ROC curve of the model\n",
+    "%pip install scikit-learn -q"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "da9a2569-03ae-4ead-a16c-e9e6ef5cf5f5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-07-12 09:50:59.928040: I tensorflow/core/util/port.cc:110] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.\n",
+      "2023-07-12 09:50:59.957734: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.\n",
+      "2023-07-12 09:51:00.083337: I tensorflow/tsl/cuda/cudart_stub.cc:28] Could not find cuda drivers on your machine, GPU will not be used.\n",
+      "2023-07-12 09:51:00.084146: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.\n",
+      "To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.\n",
+      "2023-07-12 09:51:00.880640: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import datetime\n",
+    "import concurrent.futures\n",
+    "\n",
+    "import pandas as pd\n",
+    "import temporian as tp\n",
+    "import matplotlib.pyplot as plt\n",
```

Comment:
As above, split the imports into those you actually need and those you only need for plotting / evaluation (plt, sns and sk_metrics). Otherwise it looks like you're creating a lot of dependencies.

Reply:
Good point.
*Done